subprojects/logging/src/main/java/org/gradle/internal/logging/serializer/LogEventSerializer.java

```diff
@@ -18,24 +18,28 @@
 
 import org.gradle.api.logging.LogLevel;
 import org.gradle.internal.logging.events.LogEvent;
+import org.gradle.internal.logging.events.OperationIdentifier;
 import org.gradle.internal.serialize.Decoder;
 import org.gradle.internal.serialize.Encoder;
 import org.gradle.internal.serialize.Serializer;
 
 public class LogEventSerializer implements Serializer<LogEvent> {
     private final Serializer<Throwable> throwableSerializer;
     private final Serializer<LogLevel> logLevelSerializer;
+    private final Serializer<OperationIdentifier> operationIdSerializer;
 
-    public LogEventSerializer(Serializer<LogLevel> logLevelSerializer, Serializer<Throwable> throwableSerializer) {
+    public LogEventSerializer(Serializer<LogLevel> logLevelSerializer, Serializer<Throwable> throwableSerializer, Serializer<OperationIdentifier> operationIdSerializer) {
```

Comment:
This class could probably use some simple test coverage - there's not any now, but it would be better if we added something basic to ensure against regressions.